packages/core/src/Cardano/Address/ByronAddress.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-bitwise */
+import { Address, AddressProps, AddressType, ByronAddressContent, ByronAddressType, ByronAttributes } from './Address';
```

Comment:
Any associated types, such as `ByronAddressContent` or `ByronAddressType` should be in the relevant file. So those ones can be here, and imported elsewhere if needed.